src/has.ts

```diff
@@ -64,6 +64,7 @@ add('dom-mutationobserver', function(): boolean {
 add('microtasks', function () {
 	return has('promise') || has('host-node') || has('dom-mutationobserver');
 });
+add('object-assign', typeof (<any> Object).assign === 'function');
```

Comment:
Yeah, it wasn't specifically directed at you, but more the wider bit reminding that we didn't close it off... I guess we can go back later and clean that up.